patches/third_party-blink-renderer-core-layout-intersection_geometry.cc.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/third_party/blink/renderer/core/layout/intersection_geometry.cc b/third_party/blink/renderer/core/layout/intersection_geometry.cc
-index 2d4fa917d1ad75b19863a02f8d3906b70b34e6f1..03f18a89117f2ef2552aa69083fcb4b15cce2ea1 100644
---- a/third_party/blink/renderer/core/layout/intersection_geometry.cc
-+++ b/third_party/blink/renderer/core/layout/intersection_geometry.cc
-@@ -163,7 +163,7 @@ void IntersectionGeometry::ClipToRoot() {
-   // date, so we fall back to the slow path. If we're unthrottled, then ensure
-   // that prepaint has run and the frame view doesn't need a paint property
-   // update.
--#if DCHECK_IS_ON()
```

Comment:
can we remove commit  https://github.com/brave/brave-core/pull/736/commits/1fd3fe869203733a4175b22a5c788f5e6695cec5
instead of adding e7e97f1328b4828e0f4f61d5bdf71b280cd0ebc0
when PR is ready to merge?

Reply:
@darkdh Yes! I assumed all these commits were going to be squashed into a single commit before the merge into master, as is our usual practice.

Reply:
We should definitely keep a few commits- would be best to break them into logical chunks IMO. It can help the blame be more useful (since you'd get a variety of comments about the intention of the commit, instead of just one)

Reply:
wait- you're probably talking about just merging these 4 into 1 🤦‍♂️ Nothing to see here

Reply:
best practice would be preserving the commits that tells a story.